src/mapgen_v6.h

```diff
@@ -38,9 +44,13 @@ extern FlagDesc flagdesc_mapgen_v6[];
 enum BiomeV6Type
 {
 	BT_NORMAL,
-	BT_DESERT
+	BT_DESERT,
+	BT_JUNGLE,
+	BT_TUNDRA,
+	BT_TAIGA,
```

Comment:
Hrmm, if this hard-coded biome mechanism is expanded too much, we risk having two competing and conflicting systems that ultimately do the same thing.
I think it's okay in this particular case because there were already hardcoded biomes, but this might become confusing to people that v6 biomes are different from biomes everywhere else.